tests/main/lxd/task.yaml

```diff
@@ -31,6 +31,9 @@ restore: |
     lxd.lxc stop my-ubuntu --force
     lxd.lxc delete my-ubuntu
 
+    # LXD alters the configuration of the cpuset cgroup. Restore the original state.
+    echo 0 > /sys/fs/cgroup/cpuset/cgroup.clone_children
```

Comment:
Should we rather save the current value in prepare and restore it here?

Reply:
I was thinking about it but it's off by default so YAGNI. The monitoring tool will tell us if this is wrong.